parcellab/common/templates/_poddisruptionbudget.tpl

```diff
@@ -11,13 +11,14 @@
 {{- define "common.poddisruptionbudget" -}}
 {{- $podDisruptionBudget := default (dict "enabled" false) .podDisruptionBudget -}}
 {{- $fullname := default (include "common.fullname" .) .name -}}
-{{- if or .Values.podDisruptionBudget.enabled $podDisruptionBudget.enabled }}
+{{- if and $podDisruptionBudget.enabled (or .Values.podDisruptionBudget.enabled $podDisruptionBudget.enabled) }}
```

Comment:
I have to admit, that I totally don't get why this broke with more than 2 extra services, and why we need this double check.

Reply:
this is not needed, rolled back